### PR TITLE
Update PR template checklist

### PR DESCRIPTION
**Pull request by tabathadelane**
We are updating our contributor quickstart PR template checklist.

### CODE REVIEW

**Review by mickeyryan42 — DISMISSED** [review]
I think the templates great! I don't know much about the goal of the ticket so this is maybe more a question.  It mentions updating the documentation to inform contributors we're enforcing the template.  I see you did that here, but is there somewhere else in the docs that we want to do that as well?

The template looks great so approving, but just thought worth mentioning.

**Review by aswanson-nr — COMMENTED** [review]
Let's get the summary placeholder text update, something like
```
A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this PR is ready for review.
```

**Comment by tabathadelane** [comment]
> Let's get the summary placeholder text update, something like
> 
> ```
> A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this PR is ready for review.
> ```

Good catch. I totally spaced that note from the zoom.

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
 # Summary
 
-A concise description of the changes being introduced. If you are submitting a PR for a
-new quickstart please review the pre merge checklist section to validate this PR is ready for review.
+A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this PR is ready for review.
```
Comment (josephgregoryii) [inline .github/pull_request_template.md:3]: After discussing, we should also add in somewhere that if a PR is not ready for review, it should remain in a draft until all parties are ready for review

Comment (aswanson-nr) [reply .github/pull_request_template.md:3]:
```suggestion
A concise description of the changes being introduced. Please review the pre-merge checklist section to validate this pull request is ready for review and merge. If it is not ready, please mark the pull request as a draft.
```

**Review by aswanson-nr — APPROVED** [review]

**Pull request merged**